.gitlab-ci-check-golang-acceptance.yml

```diff
@@ -0,0 +1,110 @@
+# .gitlab-ci-check-golang-acceptance.yml
+#
+# This gitlab-ci template build, runs and publises into codecov acceptance
+# tests from a Go repository.
+#
+# It assumes a tests/ folder with a Dockerfile for the Python test runner,
+# and a Dockerfile.acceptance-testing for the Go test binary.
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-golang-acceptance.yml'
+#
+# The template uses latest golang version. To override the version
+# to use, append to your .gitlab-ci.yml:
+# test:build_acceptance:
+#   image: golang:1.11
+#
+# Requires the following variables set in the project CI/CD settings:
+# CODECOV_TOKEN: Token from codecov.io for this repository
+#
+
+stages:
+  - test_prep_build
+  - test_prep
+  - test
+  - publish
+
+test:build_acceptance:
+  stage: test_prep_build
+  image: golang:1.13
+  before_script:
+    # Prepare GOPATH for the build
+    - mkdir -p /go/src/github.com/mendersoftware
+    - cp -r ${CI_PROJECT_DIR} /go/src/github.com/mendersoftware/${CI_PROJECT_NAME}
+    - cd /go/src/github.com/mendersoftware/${CI_PROJECT_NAME}
+  script:
+    - CGO_ENABLED=0 go test -c -o ${CI_PROJECT_NAME}-test -coverpkg $(go list ./... | grep -v vendor | grep -v mock | grep -v test | tr  '\n' ,);
+    - cp ${CI_PROJECT_NAME}-test ${CI_PROJECT_DIR}
+  artifacts:
+    untracked: true
+    paths:
+      - ${CI_PROJECT_NAME}-test
+
+test:prepare_acceptance:
+  stage: test_prep
+  image: docker
+  services:
+    - docker:dind
+  dependencies:
+    - test:build_acceptance
+  before_script:
+    - export DOCKER_REPOSITORY="mendersoftware/${CI_PROJECT_NAME}"
+    - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
+    - export SERVICE_IMAGE=$DOCKER_REPOSITORY:$DOCKER_TAG
+    - export COMMIT_TAG="$CI_COMMIT_REF_SLUG"_"$CI_COMMIT_SHA"
```

Comment:
Nitpick: 
Simply quote the whole thing mby?

Reply:
Okey. I'll fix in `.gitlab-ci-check-golang-dockerimage.yml` also.